src/api/db/data/20250131094818_nullify_targets_on_bs_request_actions.rb

```diff
@@ -4,14 +4,17 @@ class NullifyTargetsOnBsRequestActions < ActiveRecord::Migration[7.0]
   # rubocop:disable Rails/SkipsModelValidations
   def up
     BsRequestAction
-      .joins('LEFT JOIN projects ON bs_request_actions.target_project_id = projects.id WHERE bs_request_actions.target_project_id IS NOT NULL AND projects.id IS NULL')
+      .left_joins(:target_project_object)
```

Comment:
Instead of the table name, we need to use the association name: [this one](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/bs_request_action.rb#L20)